v22.1/ui-queues-dashboard.md

```diff
@@ -0,0 +1,189 @@
+Time Series Maintenance | The number of replicas which failed processing in the time series maintenance queue, as tracked by the `queue.tsmaintenance.process.failure` metric.
+
+## Queue Processing Times
+
+<img src="{{ 'images/v22.1/ui_queue_time.png' | relative_url }}" alt="DB Console queue processing time graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Queue Processing Times** graph shows you the processing rate across various queue systems.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+GC | The processing rate for the garbage collection queue, as tracked by the `queue.gc.processingnanos` metric.
+Replica GC | The processing rate for the replica garbage collection queue, as tracked by the `queue.replicagc.processingnanos` metric.
+Replication | The processing rate for the replication queue, as tracked by the `queue.replicate.processingnanos` metric.
+Split | The processing rate for the split queue, as tracked by the `queue.split.processingnanos` metric.
+Consistency | The processing rate for the consistency checker queue, as tracked by the `queue.consistency.processingnanos` metric.
+Raft Log | The processing rate for the Raft log queue, as tracked by the `queue.raftlog.processingnanos` metric.
+Raft Snapshot | The processing rate for the Raft repair queue, as tracked by the `queue.raftsnapshot.processingnanos` metric.
+Time Series Maintenance | The processing rate for the time series maintenance queue, as tracked by the `queue.tsmaintenance.processingnanos` metric.
+
+## Replica GC Queue
+
+<img src="{{ 'images/v22.1/ui_replicagc_queue.png' | relative_url }}" alt="DB Console replica GC queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Replica GC Queue** graph shows you various details about the health and performance of the replica garbage collection queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the replica garbage collection queue, as tracked by the `queue.replicagc.process.success` metric.
+Pending Actions | The number of pending replicas in the replica garbage collection queue, as tracked by the `queue.replicagc.pending` metric.
+Replicas Removed | The number of replica removals attempted in the replica garbage collection queue, as tracked by the `queue.replicagc.removereplica` metric.
+
+## Replication Queue
+
+<img src="{{ 'images/v22.1/ui_replication_queue.png' | relative_url }}" alt="DB Console replication queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Replication Queue** graph shows you various details about the health and performance of the replication queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
```

Comment:
Same suggestion as the previous table, might be more readable with the full metric name split into its own column so they are aligned vertically.